webapp/src/component/security/OAuthService.tsx

```diff
@@ -58,13 +58,15 @@ export const oauth2Service = (
   const redirectUri = LINKS.OAUTH_RESPONSE.buildWithOrigin({
     [PARAMS.SERVICE_TYPE]: 'oauth2',
   });
+  const authUrl = new URL(authorizationUrl);
+  authUrl.searchParams.set('client_id', clientId);
+  authUrl.searchParams.set('redirect_uri', redirectUri);
+  authUrl.searchParams.set('response_type', 'code');
+  authUrl.searchParams.set('scope', scopes.map((scope) => `${scope}`).join('+'));
```

Comment:
Actually i need to look at this line a little bit ... because using the `+` here we are more/less manually doing an encode on this parameter, so when `URL.toString` runs those are then getting encoded as `%2B` (a literal plus rather than a url encoded space)

Reply:
I've updated it to `.join(' ')`, so it will get encoded properly as a space. Note: It will be encoded as a `%20` instead of a `+`. But technically `%20` is preferred from a compatibility stand point.